src/main/java/tech/gruppone/stalker/server/model/db/UserDao.java

```diff
@@ -0,0 +1,23 @@
+package tech.gruppone.stalker.server.model.db;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+import org.influxdb.annotation.Column;
```

Comment:
occhio che questa è l'annotation sbagliata! ci sono due column, uno di influxdb e uno di spring data:
```suggestion
import org.springframework.data.relational.core.mapping.Column;
```